views/stanford/about_get_involved.pug

```diff
@@ -70,55 +167,5 @@ block content
                   h4.media-heading= _("Email Us")
 
-            //-div.col-lg-6.col-md-6
-              a(href='https://stanford-oval.slack.com').media
-                div.media-left.media-middle
-                  img(src=Config.ASSET_CDN + '/images/logo-slack.png').media-object
-                div.media-body.media-middle
-                  h4.media-heading= _("Slack (by invitation only)")
-
-      section.divider#section-what-can-i-do-for-almond
-        h3= _("What can I do for Almond?")
-
-        div.container
-          //-div.row
-            div.col-lg-offset-2.col-lg-offset-8
-              blockquote
-                p Ask not what Almond can do for you — ask what you can do for Almond.
-                footer J. F. K., potentially.
-
-          div.row
-            div.col-md-6
-              h4= _("Contribute To Thingpedia")
-              p.
-                Do you own a device, or know of a service that you want to control through Almond?
-                You can add it to Thingpedia, and have Almond immediately learn about it!
-              p.
-                Check out our #[a(href='/doc/thingpedia-intro.md') developer documentation],
-                and when you're ready, #[a(href='/user/request-developer') apply to our developer program].
-
-            div.col-md-6
-              h4= _("Develop Almond")
-              p.
-                Have you found a bug? Or a feature that you really want? Head to our
-                #[a(href='https://github.com/stanford-oval') Github page] to find our
-                code base. Fork it, patch it, grow it.
-              p.
-                Also, don't hesitate to get in touch through our
-                #[a(href='https://community.almond.stanford.edu') support forum]
-                for help with larger projects, access to internal documentation,
-                and details of the Almond architecture.
-
-            //-div.col-md-6
-              h4= _("Donate to Almond")
-              p.
-                Almond is generously funded by the
-                #[a(href='https://mobisocial.stanford.edu') Mobile and Social Computing Lab], an industrial affiliates
-                program within Stanford University's #[a(href='https://engineering.stanford.edu/') School of Engineering].
-              p.
-                The #[a(href='https://mobisocial.stanford.edu/index.php?page=affiliates') affiliates program]
-                provides an opportunity for industry and other organization to collaborate and interact
-                with our research team, through a recurrent sponsorship agreement.
-                Smaller donations can also be provided #[a(href='https://engineering.stanford.edu/get-involved/give') to the university].
-                All contributions to Stanford are tax deductible in the United States.
-
+  script(src='https://cdn.jsdelivr.net/npm/@glidejs/glide')
+  script(src=Config.ASSET_CDN + '/javascripts/get-involved.js')
```

Comment:
Is there a reason to put this JS here instead of the "scripts" pug block? If the problem is blocking the load, we can mark them `async=true`.

Reply:
Yeah. It needs to be loaded after everything, I think. Otherwise the carousel doesn't work. It's hacky unfortunately

Reply:
If you wrap everything inside get-involved.js in `$(function() { ... })` it will work (`$()` adds a DOMContentLoaded event handler)

Reply:
Ah thanks. Fixed.